src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url("../images/star.svg");
+    background-position: center;
+    background-repeat: no-repeat;
+    width: $star-size;
+    height: $star-size;
+
+    &:not(:first-child) {
+      margin-left: $star-size / 4;
+    }
+  }
+
+  &--4 .stars__star:nth-child(-n + 4) {
+    background-image: url("../images/star-active.svg");
+  }
```

Comment:
Make sure to reuse part of [Stars task](https://github.com/mate-academy/layout_stars) here, and keep it operational in case you change block modifier.